tests/handlers/test_estimandizer.py

```diff
@@ -27,24 +26,25 @@ def test_check_and_create_estimands_historical(va_governor_county_data):
     estimandizer = Estimandizer()
     (output_df, result_columns) = estimandizer.check_and_create_estimands(va_data_copy, estimands, True)
 
-    assert "baseline_party_vote_share_dem" in output_df.columns
     assert "results_party_vote_share_dem" in output_df.columns
     assert result_columns == ["results_party_vote_share_dem"]
+    assert output_df["results_party_vote_share_dem"].isnull().all()
```

Comment:
what is this testing?

Reply:
When we're working with a historical election and calling (the soon-to-be-renamed) `check_and_create_estimands()`, `results_estimand` columns are being set to `nan`.  This is to make sure that happens for this new custom estimand we've added 🤔